Microsoft.DotNet.Interactive.Jupyter.Tests/ExecuteRequestHandlerTests.cs

```diff
@@ -107,7 +111,7 @@ public async Task sends_ExecuteReply_message_when_submission_contains_only_a_dir
                 new CSharpKernel()
             };
 
-            var handler = new ExecuteRequestHandler(kernel);
+            var handler = new ExecuteRequestHandler(kernel, CurrentThreadScheduler.Instance);
```

Comment:
Boilerplate changes to the tests make me wonder if the class should just default to this?